src/component/bridge.rs

```diff
@@ -124,3 +152,52 @@ impl<C: StatefulComponent> Bridge<C, C::Root> {
         }
     }
 }
+
+/// Messages being forwarded into the a component's main service.
+enum InnerMessage<Input, CmdOutput> {
+    ModelUpdate(Input),
+    CommandUpdate(CmdOutput),
+}
```

Comment:
This is necessary because the model must be captured by only one closure, right? It might be nice to leave a note for anyone who reads the code so nobody wonders why it doesn't simply listen on two receivers.

Reply:
Yea, I can add the note.